Modules/System/Azure Blob Services API/src/Helper/ABSOptionalParameters.Codeunit.al

```diff
@@ -331,4 +367,7 @@ codeunit 9047 "ABS Optional Parameters"
         ABSFormatHelper: Codeunit "ABS Format Helper";
         RequestHeaders: Dictionary of [Text, Text];
         Parameters: Dictionary of [Text, Text];
+        NeedToSpecifyHeaderErr: Label 'You need to specify the "%1"-header to set this header.', Comment = '%1 = HttpHeader identifier';
```

Comment:
These strings appear to not be used anymore.

Reply:
Yes, I guess you're right. This was used in one the removed functions. Could you remove it? Or should I push another commit?

Reply:
I'll have them removed :)